client/rest-high-level/src/test/java/org/opensearch/client/OpenSearchRestHighLevelClientTestCase.java

```diff
@@ -84,6 +85,7 @@
 import static org.hamcrest.Matchers.lessThan;
 import static org.hamcrest.Matchers.notNullValue;
 
+@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/pull/2683")
```

Comment:
Thank you for raising this PR.

I have one concern that muting test classes has potential of introducing bugs from other code changes exercising these two test classes. How do we have plan to cover this ?

Reply:
Hi @dreamer-89 thank you for reviewing my PR.
My thought is to hold merging this PR until all the other PRs are ready to be merged in #2511. When getting ready, then merging all those PRs together in a short time. The potential of other code changes merged can be reduced by try to shorten the interval of muting the tests.
This is a good point, I should put up some notice saying not to merge this PR in a hurry. 😄

Reply:
Thanks @tlfeng for clarifying. 

Is it possible we can add `OpenSearchRestTestCase.expectWarnings` in tests in above two tests files wherever we are adding `master_timeout`. I think this is how we are tackling other deprecated features on client side.

Reply:
Thank you for your good suggestion!
You are right. 👍 Looks like I should turn off the `enableWarningsCheck` https://github.com/opensearch-project/OpenSearch/blob/1.3.1/test/framework/src/main/java/org/opensearch/test/OpenSearchTestCase.java#L407 for the affected test classes. 
There are 24 classes extends this class `OpenSearchRestHighLevelClientTestCase` (and one class has turned it off https://github.com/opensearch-project/OpenSearch/blob/1.3.1/client/rest-high-level/src/test/java/org/opensearch/client/BulkProcessorIT.java#L89) so the amount of work is reasonable.
I will apply it to the affected test classes.

Reply:
Updated the code. The `enableWarningsCheck` doesn't work for this case, but instead, I found using `setStrictDeprecationMode(false)` to build the RestClient can make the test allow deprecation warnings. This will be a safer way 😄.